dqc_us_rules/dqc_us_0001.py

```diff
@@ -51,16 +53,18 @@ def filter_func(model_object):
                 )
 
             for axis in filter(filter_func, relset.fromModelObjects()):
-                _run_axis_checks(axis, axis_config, relset, val, role)
+                _run_axis_checks(axis, axis_key, axis_config, relset, val, role, checked_axes)
```

Comment:
I wonder if this might work fine if checked_axes uses axis itself as the key instead of axis_key to keep the signature small.